README.md

```diff
@@ -107,7 +107,7 @@ mdsf init
 | TypeScript | `biome`, `deno_fmt`, `prettier`                               |
 | Vue        | `prettier`                                                    |
 | Xml        | `xmlformat`, `xmllint`                                        |
-| Yaml       | `prettier`, `yamlfmt`                                         |
+| Yaml       | `prettier`, `yamlfix`, `yamlfmt`                              |
```

Comment:
at some point you should have a page with the supported linter and where to find them to install them

"prettier" is very common know, but yamlfix vs yamlfmt

I'm talking about a markdown page with the link of the linter

Please note it could be simply link in this table, it's up to you.

And yes, I know once again, it's not directly related to the change in this MR

Reply:
I just used the same commands as when I installed locally 😛

Reply:
Definitely, my plan was to add a table like this:

| Formatter                                      | Languages | Description                                                   |
| ---------------------------------------------- | --------- | ------------------------------------------------------------- |
| [yamlfix](https://github.com/lyz-code/yamlfix) | yaml      | A simple opinionated yaml formatter that keeps your comments! |

Just haven't gotten around to updating [scripts/update-supported-languages.mjs](scripts/update-supported-languages.mjs) 😄

Reply:
https://github.com/hougesen/mdsf/issues/172 was opened